dot/telemetry/telemetry.go

```diff
@@ -23,6 +23,8 @@ const (
 	txPoolImportMsg = "txpool.import"
 )
 
+//go:generate mockgen -destination=mock_telemetry_client_test.go -package=$GOPACKAGE . Client
```

Comment:
Where is this mock used in the telemetry package? 🤔

Reply:
Not sure, I've removed this.